src/pnpq/devices/utils.py

```diff
@@ -0,0 +1,28 @@
+from serial.tools.list_ports import comports as list_comports
+import logging
+
+logger = logging.getLogger("utils")
+
+USB_HUB_iVENDOR = "2109"
```

Comment:
`USB_HUB_VENDOR_ID` is fine. I don't want to put `i` in the constant name.

Reply:
AVAILABLE_USB_HUBS was defined to be a list of (vendor, product) pairs.